### PR TITLE
CASS-18228 Reorganize doc files to new Info Arch standard

### PR DESCRIPTION
**Pull request by polandll**
CHANGES TO STRUCTURE

mkdir overview
mv getting_started getting-started
cd developing; mv data_modeling data-modeling
cd architecture; mv storage_engine.adoc storage-engine.adoc
mkdir managing
mkdir securing
mkdir tooling
mkdir references
mv operating/ managing/
mkdir developing
mv cql developing/
mv data_modeling/ developing/
mv faq overview/
mv tools managing/
mv configuration/ managing/

cd tooling
touch generate-tokens
touch hash-password-tool
touch cass-stress-deprecated

mkdir integrating
mv plugins/ integrating/
mv faq overview

mv new/virtualtables.adoc managing/
mv getting-started/java11.adoc references/

Added files to "new" section as placeholders for documents to be written.

### CODE REVIEW

File: doc/modules/cassandra/pages/new/4.2-new.adoc
Comment (michaelsembwever) [inline doc/modules/cassandra/pages/new/4.2-new.adoc:1]: the files in this directory were originally just `index.adoc` (and are referenced from above as such).

they are versioned, there's no need to have them as separate docs… 🤷🏻‍♀️

File: doc/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ to do so by submitting your contribution like any other patch following
 [cols="a,a"]
 |===
 
-| xref:cassandra:getting_started/index.adoc[Getting started] | Newbie starting point
+| xref:cassandra:getting-started/index.adoc[Getting started] | Newbie starting point
 
 | xref:cassandra:new/index.adoc[What's new in 5.0] | What's new in Cassandra 5.0
```
Comment (michaelsembwever) [inline doc/modules/ROOT/pages/index.adoc:20]: does this work anymore?

**Comment by michaelsembwever** [comment]
How will "Developing" in the new structure (docs for users, vs operators) distinguish itself from "Development" (developers working on the project itself) ?

**Comment by polandll** [comment]
The Developing section is all about CQL in the case of Apache C*. 
Examples of how to interact with the database are the meat of this section. 

-------- Original message --------

File: doc/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (polandll) [reply doc/modules/ROOT/pages/index.adoc:20]: I've fixed this, but I've created a new PR.

File: doc/modules/cassandra/pages/new/4.2-new.adoc
Comment (polandll) [reply doc/modules/cassandra/pages/new/4.2-new.adoc:1]: I have these different versions in separate files, to make adding and deleting easier.

**Comment by polandll** [comment]
CLosing, due to too many conflicts. Opened new PR.

**Pull request closed without merging**